docs/deployment/gae_flex_datastore.rst

```diff
@@ -0,0 +1,167 @@
+.. _appengine_flexible_tutorial:
+
+Google App Engine Flexible with Datastore and :app:`Pyramid`
+============================================================
+
+It is possible to run a :app:`Pyramid` application on `Google App Engine <https://cloud.google.com/appengine/>`_.  This tutorial is written "environement agnostic" meaning the commands here should work on Linux, Mac or Windows. This tutorial also assumes you've already installed and created a :app:`Pyramid` application, and that you have a Google App Engine account.
```

Comment:
- Please use one line per sentence for better diffs.
- s/"environement agnostic" meaning/"environment agnostic", meaning
- s/Mac or/macOS, or